multi-git-dashboard/src/components/cards/ProjectManagementCard.tsx

```diff
@@ -130,93 +228,164 @@ const ProjectManagementCard: React.FC<ProjectManagementCardProps> = ({
         >
           {sortedAssigneeStatsArrays.map((assigneeStatsArray, index) => (
             <Carousel.Slide key={index}>
-              <Text
-                size="sm"
-                fw={500}
-                style={{ textAlign: 'center', marginBottom: 8 }}
+              <Group
+                style={{
+                  paddingLeft: '6%',
+                  paddingRight: '6%',
+                  paddingTop: '2%',
+                }}
               >
-                Sprint ending {new Date(sortedKeys[index]).toLocaleDateString()}
-              </Text>
-              <BarChart
-                h={400}
-                data={assigneeStatsArray}
-                dataKey="Assignee"
-                withLegend
-                legendProps={{ verticalAlign: 'bottom', height: 50 }}
-                series={[
-                  { name: 'Issues', color: 'blue.6' },
-                  { name: 'Story Points', color: 'teal.6' },
-                ]}
-              />
+                <Table
+                  striped
+                  highlightOnHover
+                  withTableBorder
+                  withColumnBorders
+                >
+                  <Table.Caption>
+                    Sprint ending{' '}
+                    {new Date(sortedKeys[index]).toLocaleDateString()}
+                  </Table.Caption>
+                  <Table.Thead>
+                    <Table.Tr>
+                      <Table.Th>Assignee</Table.Th>
+                      <Table.Th>Issues</Table.Th>
+                      <Table.Th>Story Points</Table.Th>
+                      <Table.Th>Story Points Per Issue</Table.Th>
+                    </Table.Tr>
+                  </Table.Thead>
+                  <Table.Tbody>{rows(assigneeStatsArray)}</Table.Tbody>
+                </Table>
+              </Group>
             </Carousel.Slide>
           ))}
         </Carousel>
       </Card>
     );
   };
 
-  const getSprintCompletionBarChart = (jiraSprints: JiraSprint[]) => {
+  const getVelocityChart = (jiraSprints: JiraSprint[]) => {
     interface SprintSummary {
```

Comment:
Can put `interfaces` at top of file for better visibility

Reply:
Moved interfaces to the top of file.